oz_tree_build/images_and_vernaculars/get_wiki_images.py

```diff
@@ -54,8 +51,8 @@
 # Copied from OZTree/OZprivate/ServerScripts/Utilities/getEOL_crops.py
 def subdir_name(doID):
     """
-    Make a valid subdirectory name in which to save images, based on the last 3 chars of
-    the data object ID
+    Make a valid subdirectory name in which to save images, based on the last
```

Comment:
That feels like an over-agressive line length. Maybe best to configure it to match what Black did, to reduce the churn?

Reply:
What did we have in `black`? The recommended is 89, but I find that too short. I'm happy to make it longer. How about e.g. 99? Or even 120?

Reply:
I see, it appears that black was simply not doing anything to comment lines regardless of the length. Going to 120 sgtm. This should in practice revert some of the Ruff changes, like in this particular case.

Reply:
Good point. If you like, I can rebase the PR and make the ruff changes 120 lines instead? I'll need to force push it all though

Reply:
Force pushing on a dev branch is fine. It's only bad on main. Or I suppose you could reconfigure the formatter to 120 and get it in as a new commit.

I don't want to make it more complicated for you than it has to be. If needed, I can just ignore comments when reviewing. It's the code logic that needs careful review.

Reply:
Thanks!